core.cpp

```diff
@@ -107,6 +107,11 @@ int Mat_Type(Mat m) {
     return m->type();
 }
 
+// Mat_Step returns size of aligned image row in bytes.
```

Comment:
I did not understand this doc, so I googled a bit and found this:

> step returns the number of bytes each matrix row occupies.

Is it the same idea?

Reply:
I changed docs, it was a c/p from include file, just from wrong place.